cypress/e2e/signIn.cy.js

```diff
@@ -2,10 +2,43 @@
 
 describe('Sign In page', () => {
   beforeEach(() => {
+    cy.visit('https://the-internet.herokuapp.com/login');
+  });
 
+  it('should provide an ability to login with existing data', () => {
+    cy.get('[name="username"]')
+      .type('tomsmith');
```

Comment:
```suggestion
      .type('tomsmith');
```
Define variables for username and password, because you use it more than 2 times

Reply:
Done.